spec/api/commentable_spec.rb

```diff
@@ -32,6 +32,11 @@ def thread_result(commentable_id, params={})
         threads.index{|c| c["body"] == "can anyone help me?"}.should_not be_nil
         threads.index{|c| c["body"] == "it is unsolvable"}.should_not be_nil
       end
+      it "returns standalone threads if explicitly requested" do
+        threads = thread_result "question_1", context: "standalone"
```

Comment:
Yes, for any context other than "course" the context must be explicitly specified in the request. The work in https://github.com/edx/edx-platform/pull/8982 adds support for passing a context parameter on an inline discussion request through to the comments service, but more work is needed in the future story to specify a context for an inline discussion module.